lib/galaxy/model/__init__.py

```diff
@@ -3669,6 +3690,9 @@ def __init__(self, type, quota):
         assert type in self.types.__members__.values(), "Invalid type"
         self.type = type
         self.quota = quota
+        # Safeguard: self was implicitly merged into this Session prior to SQLAlchemy 2.0.
```

Comment:
since these are always the same and we may want to eliminate those patterns: can you move this into a utility function so we can disable this in one go later on ?

Reply:
sure! I considered this, but opted to keep the verbose version so it's not mistaken for a permanent feature. I'll make it function.